### PR TITLE
Color of FavIcon in the footer is changed to keppel green

### LINKED ISSUE #9022: Change heart color to Keppel green in dark mode
### Current Behavior
The heart in the footer in Meshery UI is in gray color in dark mode

image

### Desired Behavior
Change the color of the heart in the footer in Meshery UI to Keppel green

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by VivekBaldha0712**
**Notes for Reviewers**

This PR fixes #9022 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/pages/_app.js [hunk 1]
```diff
@@ -594,8 +594,12 @@ class MesheryApp extends App {
                         ) : (
                           <>
                             {' '}
-                            Built with <FavoriteIcon className={classes.footerIcon} /> by the Layer5
-                            Community
+                            Built with{' '}
+                            <FavoriteIcon
+                              style={{ color: '#00ab84' }}
```
Comment (leecalcote) [inline ui/pages/_app.js:599]: Thanks for this. Wrong color, sir.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/pages/_app.js (hunk 1 above)
Comment (leecalcote) [reply ui/pages/_app.js:599]: Please use a theme-based reference to the "#00b39f" (Keppel green) variable.

Comment (VivekBaldha0712) [reply ui/pages/_app.js]: I have done the required changes suggested by you. Please have a look

Comment (sudhanshutech) [reply ui/pages/_app.js]: @VivekBaldha0712 theme reference means don't use directly the hex colors , we have defined theme names for colors under [`themes/app.js`](https://github.com/meshery/meshery/blob/master/ui/themes/app.js) from use a reference called success naming like `theme.palette.seconadary.success`

File: ui/pages/_app.js [hunk 2]
```diff
@@ -594,8 +594,12 @@ class MesheryApp extends App {
                         ) : (
                           <>
                             {' '}
-                            Built with <FavoriteIcon className={classes.footerIcon} /> by the Layer5
-                            Community
+                            Built with{' '}
+                            <FavoriteIcon
+                              style={{ color: '#00b39f' }}
```
Comment (Yashsharma1911) [inline ui/pages/_app.js:599]:
```suggestion
                              style={{ color: this.state.theme === 'dark' ? theme.palette.secondary.focused : '' }}
```

@VivekBaldha0712 this is what requested, please test this change if works update the PR

**Review by Yashsharma1911 — CHANGES_REQUESTED** [review]

File: ui/pages/_app.js (hunk 2 above)
Comment (Yashsharma1911) [reply ui/pages/_app.js:599]: It would be helpful if you can add screen shot after the changes

**Comment by saurabh100ni** [comment]
Thanks for raising the PR, let's Discuss this in the Meshery Development Call today,
Please add this as an agenda in the Doc  : [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)

File: ui/pages/_app.js (hunk 2 above)
Comment (leecalcote) [reply ui/pages/_app.js:542]: @VivekBaldha0712 ... a screenshot of the UI, not the code.

Comment (VivekBaldha0712) [reply ui/pages/_app.js:542]: Sure. Here it is...
image

Comment (Yashsharma1911) [reply ui/pages/_app.js:542]: Thankyou for adding the screenshot please make sure that we only change color to `#00b396` in dark mode only

**Comment by abhijeetgauravm** [comment]
Let's discuss this on Meshery Development Meeting today at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#).

**Review by leecalcote — APPROVED** [review]
Thanks, @VivekBaldha0712

**Pull request merged**